bundles/org.openhab.binding.chatgpt/src/main/java/org/openhab/binding/chatgpt/internal/ChatGPTConfiguration.java

```diff
@@ -25,4 +25,12 @@ public class ChatGPTConfiguration {
     public String apiKey = "";
     public String apiUrl = "https://api.openai.com/v1/chat/completions";
     public String modelUrl = "https://api.openai.com/v1/models";
+    public boolean useSemanticModel = false;
```

Comment:
Make it the same as in the readme
```suggestion
    public boolean useSemanticModel = true;
```